NitroxClient/Communication/Packets/Processors/DiscordRequestIPProcessor.cs

```diff
@@ -1,13 +1,33 @@
-﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+﻿using System.Net;
+using System.Threading.Tasks;
+using NitroxClient.Communication.Packets.Processors.Abstract;
 using NitroxClient.MonoBehaviours.Discord;
+using NitroxModel.Helper;
 using NitroxModel.Packets;
 
 namespace NitroxClient.Communication.Packets.Processors;
 
 public class DiscordRequestIPProcessor : ClientPacketProcessor<DiscordRequestIP>
 {
-    public override void Process(DiscordRequestIP packet)
+    public override async void Process(DiscordRequestIP packet)
```

Comment:
Is this async method correctly run/handled from our framework? If not we could just Use Task.Run inside of it, no?

Reply:
It seems to run correctly and finds my WAN IP. So I *think* we can keep as is, if you're happy